Sources/MapboxNavigation/SpeedLimitView.swift

```diff
@@ -1,6 +1,14 @@
 import UIKit
 import MapboxDirections
 
+/**
+ Indicates which view mode to display, the current speed or the speed limit.
+ */
+public enum SpeedViewMode {
```

Comment:
should we be introducing these properties in this pr? once it is merged, it could break backwards compatibility if we wanted to change them.